team_picker/index.js

```diff
@@ -0,0 +1,68 @@
+var express = require('express')
+var app = express()
+
+const methodOverride = require("method-override")
+
+app.use(express.urlencoded({extended: true}))
+
+// ---------------Method Overrride Middleware--------------
+app.use(methodOverride((req, res) => {
+    if(req.body && req.body._method) {
+        const method = req.body._method;
+        return method
+    }
+}))
+
+
+const path = require("path")
+
+app.use(express.static(path.join(__dirname, 'public')));
+
+
+const logger = require('morgan');
+app.use(logger('dev'));
+
+
+
+// app.get('/', (req, res) => {
```

Comment:
You need this root route otherwise `localhost:3000` errors out with 404 on load. Always define this even if you display nothing on the page though technically you should since it's a home page. If you don't want to render anything in this route you can redirect it to the index page from here.